src/views/ConfigurationAlertsView.vue

```diff
@@ -6,12 +6,24 @@
         class="flex flex-col justify-around align-start ml-5 max-h-[85vh] overflow-y-auto"
         :class="interfaceStore.isOnSmallScreen ? 'max-w-[70vw]' : 'max-w-[40vw]'"
       >
-        <v-switch
-          v-model="alertStore.enableVoiceAlerts"
-          label="Enable voice alerts"
-          color="white"
-          class="mt-2 -mb-2 ml-3"
-        />
+        <div class="flex w-full justify-between pr-10">
+          <v-switch
+            v-model="alertStore.enableVoiceAlerts"
+            label="Enable voice alerts"
+            color="white"
+            class="mt-2 -mb-2 ml-3"
+          />
+          <v-checkbox
+            v-model="alertStore.enabledAlertLevels.find((level) => level.level === AlertLevel.Critical)!.enabled"
+            v-tooltip="
+              'Critical system alerts work separately from your voice alert settings. These alerts are rare but really important, so we recommend keeping them on.'
+            "
+            label="Critical system alerts"
+            hide-details
+            color="white"
+            class="mt-1"
+          />
+        </div>
```

Comment:
The problem with reusing the Critical level is that it's being also used on `MAV_SEVERITY_EMERGENCY`, `MAV_SEVERITY_ALERT` and `MAV_SEVERITY_CRITICAL`, from Ardupilot. Their names suggest that they are only thrown on very critical situations, but I have found it to not be true.

Right now, with the Critical level enabled (which I want to have for the joystick warning), I'm being spammed by "Ardupilot logger stuck thread" voice alerts, so I will have to disable the critical level, and in consequence I will lose the joystick alert, which I want to have.

I can think of three solutions here:

1. Have this joystick alert on a separate level.

2. Make each `MAV_SEVERITY` have it's own dedicated level.

3. Create a dynamic alert registration system, where each new alert will create an entry for it in the alerts menu, and the user will be able to disable each alert individually. 

Solution 1 is the easy one. Solution 2 is also easy and more long-term, but makes Cockpit more tied to Ardupilot (and we want to go in the opposite direction). Solution 3 seems to be the correct one, but takes a little more work (although not that much I think).